tests/js/spec/views/performance/transactionVitals.spec.jsx

```diff
@@ -341,5 +341,57 @@ describe('Performance > Web Vitals', function () {
         ),
       });
     });
+
+    it('renders an info alert when missing web vitals data', async function () {
+      MockApiClient.addMockResponse({
+        url: '/organizations/org-slug/events-vitals/',
+        body: {
+          'measurements.fp': {poor: 1, meh: 2, good: 3, total: 6, p75: 4567},
+          'measurements.fcp': {poor: 1, meh: 2, good: 3, total: 6, p75: 1456},
+        },
+      });
+
+      const {organization, router, routerContext} = initialize({
+        query: {
+          lcpStart: '20',
+        },
+      });
+
+      const wrapper = mountWithTheme(
+        <WrappedComponent
+          organization={organization}
+          location={router.location}
+          router={router}
+        />,
+        routerContext
+      );
+
+      await tick();
+      wrapper.update();
+
+      expect(wrapper.find('Alert')).toHaveLength(1);
```

Comment:
this is kind of minor, but maybe we can also assert on some of the content of the alert just to make the test more specific. alerts can be shown for multiple reasons so it'd be nice to test that the specific alert is shows to users

Reply:
I think it wouldn't matter here since this is the only possible alert than can be rendered on this page. I'm also asserting that no alerts are rendered if the conditions aren't met, so that should cover both cases :)

I considered asserting the text here too, but it would cause this test to be coupled with the text inside the Alert component, which is a minor nuisance if somebody wanted to change the text in the future.

Reply:
this is the only possible alert... for now :) but yeah we can deal with that when there are more alerts
yeah I agree on not making assertions on the copy 👍